README.md

```diff
@@ -1,2 +1,180 @@
+16. Custom scripts for visualization of results ([R](https://www.r-project.org/) and [various packages](https://github.com/lumc-sasc/wf-iclipseq/blob/main/CITATION.md#r-packages))
+17. Motif detection ([STREME](https://meme-suite.org/meme/doc/download.html))
+18. Summarizing results up to the last quality control step ([MultiQC](https://multiqc.info/))
+
+Steps in the pipeline (with the exception of STAR alignment, for the time being) can be skipped using the [conf/params.config](conf/params.config) file. The pipeline contains multiple quality control steps which can be used to closely monitor the data before peak calling. By default, quality control before and after trimming is enabled, while the others are disabled.
+
+*not included in the pipeline yet, but the pipeline does contain the nextflow script.
+
+## Default run
+By default, the pipeline performs the following things:
+1. QC on raw reads by FastQC
+2. Quality and adapter trimming by Trim Galore!
+3. Genome indexing by STAR
+4. Genome alignment by STAR (allowing multimappers, with `--outFilterMultimapNmax 20` and doing chimeric detection)
+5. Alignment sorting and indexing by SAMtools
+6. Deduplication by UMI-Tools
+7. Crosslink extraction
+8. Peak calling by PureCLIP
+9. Resizing PureCLIP binding sites for gene annotation by 8bp (change this to match your RBPs binding footprint, can also be deactivated)
+10. Gene annotation by Bash & BEDTools, and HOMER
+11. Motif detection by STREME
+12. Report by MultiQC
+
+### Output created by default run
+The output of a default run creates the following tree:
+```bash
+├── 01_fastqc
+├── 02_trimgalore
+│   └── fastqc
+├── 04_star
+│   ├── log
+│   ├── samtools_stats
+│   └── umitools
+│       ├── bai
+│       ├── bam
+│       └── fastqc
+├── 05_extract_crosslinks
+│   └── 06_crosslink
+│       ├── merged
+│       ├── neg
+│       └── pos
+├── 06_peak_calling
+│   └── pureclip
+├── 07_downstream_analysis
+│   ├── 00_sites_with_peakID
+│   ├── 01_bedtools_annotations
+│   ├── 01_homer_annotations
+│   └── 02_streme_motifs
+│       └── bedtools_resized
+├── 08_multiqc
+│   ├── multiqc_data
+│   └── multiqc_plots
+│       ├── pdf
+│       ├── png
+│       └── svg
+└── pipeline_info
+```
+
+If rRNA filtering is enabled, there will be a folder named `03_sortmerna/`.
+
+# Requirements
+Nextflow (>23.04.4) and Singularity are required to run this pipeline. Both can be installed using Conda. To download Conda, follow this [tutorial](https://docs.conda.io/projects/conda/en/latest/user-guide/install/linux.html). Check that Conda is up-to-date:
+
+`conda --version`
+
+```plaintext
+conda 23.9.0
+```
+
+Make a conda environment to download the packages in and activate it:
+```
+conda create -n nf_env
+conda activate nf_env
+```
+
+You can install Nextflow using [Bioconda](https://bioconda.github.io/), after setting up the proper channels: `conda install -c bioconda nextflow`
+
+Or a specific version: `conda install -c bioconda nextflow=23.10.0`
```

Comment:
For bioconda installs I believe `conda install -c conda-forge -c bioconda` command line is preferred. The Java runtime needs to come from conda-forge.